src/ansys/systemcoupling/core/__init__.py

```diff
@@ -74,6 +75,11 @@ def launch(
         ``"241"`` ("2024 R1" release), unless either of the environment
         variables ``SYSC_ROOT`` or ``AWP_ROOT`` has been set. It is considered
         to be an error if either these is set *and* ``version`` is provided.
+    start_output: bool, optional
+        Boolean to specify if the user want to stream system coupling output.
```

Comment:
typo: "...user want**s** to..."

Just say "printed to standard output." in the last sentence.